docs/api/native-image.md

```diff
@@ -137,15 +137,17 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
-### `nativeImage.createFromBuffer(buffer[, scaleFactor])`
+### `nativeImage.createFromBuffer(buffer[width, height, scaleFactor])`
```

Comment:
Also you introduce a breaking change here by moving the `scaleFactor` argument. Existing code will break!